robo4j-socket-http/src/main/java/com/robo4j/socket/http/units/ServerContext.java

```diff
@@ -42,7 +42,8 @@ public final class ServerContext implements SocketContext<ServerPathConfig> {
 	 */
 	private final Map<String, Object> properties = new HashMap<>();
 
-	ServerContext() {
+	// TODO: review context creation public modifier due to the JPMS
+	public ServerContext() {
```

Comment:
You need either a public non arg constructor or a public static constructor factory of some sort in order to be "public" API.